### DISCUSSION TITLE
useContext 에러

### DISCUSSION THREAD

**Discussion by myeongsu-ko** [velopert/dealing-with-react-native#39] [Q&A]
다이어리앱 만들기 과정을 해보고있는데 
useContext 사용 때마다 에러나 나와서.. 어떻게 해결할수있을까요..??
![Image 001](https://user-images.githubusercontent.com/71993926/194536813-0e1a22ae-c5c9-44e8-b011-e22968894ac5.png)
![Image 002](https://user-images.githubusercontent.com/71993926/194536816-ce2e68d4-2d67-4ea4-8391-d68efc4efc8e.png)
![Image 003](https://user-images.githubusercontent.com/71993926/194536820-edab5bdb-ab98-4593-baac-bbeec01ac536.png)

**Comment by KNOXINUS** [comment]
2번째 사진에서 (13번째 줄) const onCreate ...가 아니라 const {onCreate} ... 로 하시면 될거에요.
